.circleci/config.yml

```diff
@@ -220,6 +223,37 @@ jobs:
           name: Run e2e collections test
           command: |
             python -m scripts.run_e2e_tests --skip-install --skip-build --suite="collections" --prod_env
+      - run:
```

Comment:
Maybe a suite for all the dashboards? [Just a suggestion]

Reply:
sure, I'll let this run though but I'll keep this comment open.